bazel/external/quiche.BUILD

```diff
@@ -253,6 +254,48 @@ cc_library(
     deps = [":quic_platform_export"],
 )
 
+cc_library(
+    name = "epoll_server_platform",
+    testonly = 1,
+    hdrs = [
+        "quiche/epoll_server/platform/api/epoll_address_test_utils.h",
+        "quiche/epoll_server/platform/api/epoll_bug.h",
+        "quiche/epoll_server/platform/api/epoll_expect_bug.h",
+        "quiche/epoll_server/platform/api/epoll_export.h",
+        "quiche/epoll_server/platform/api/epoll_logging.h",
+        "quiche/epoll_server/platform/api/epoll_ptr_util.h",
+        "quiche/epoll_server/platform/api/epoll_test.h",
+        "quiche/epoll_server/platform/api/epoll_thread.h",
+        "quiche/epoll_server/platform/api/epoll_time.h",
+    ],
+    visibility = ["//visibility:public"],
+    deps = ["@envoy//test/extensions/quic_listeners/quiche/platform:epoll_server_platform_impl_lib"],
+)
+
+cc_library(
+    name = "epoll_server_lib",
+    testonly = 1,
+    srcs = [
+        "quiche/epoll_server/fake_simple_epoll_server.cc",
+        "quiche/epoll_server/simple_epoll_server.cc",
+    ],
+    hdrs = [
+        "quiche/epoll_server/fake_simple_epoll_server.h",
+        "quiche/epoll_server/simple_epoll_server.h",
+    ],
+    copts = envoy_copts("@envoy") + ["-Wno-error=unused-parameter"],
+    visibility = ["//visibility:public"],
+    deps = [":epoll_server_platform"],
+)
+
+envoy_cc_test(
+    name = "epoll_server_test",
```

Comment:
This test is also defined in test/extensions/quic_listeners/quiche/platform/BUILD, should we delete one of them?